app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    mv, old_file, new_file = command.split(" ")
+    if old_file == new_file:
+        return
+    path = new_file.split("/")
+    new_dir = "/".join(path[:-1])
```

Comment:
Will be better to use the function  `os.path.dirname()` to separate dir name from all path